lib/src/main/scala/spinal/lib/eda/fusesoc/FuseSocGeneratorBuilder.scala

```diff
@@ -0,0 +1,126 @@
+package spinal.lib.eda.fusesoc
+import spinal.core._
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.{YAMLFactory, YAMLGenerator}
+import com.fasterxml.jackson.module.scala._
+
+import scala.collection.immutable.Map.Map1
+import scala.collection.mutable
+import scala.reflect.ClassTag
+
+class GeneratorOutput() {
```

Comment:
It's good to have a prefix Fusesoc, it's easy to find out where it belongs when you use it.